content/docs/installation/featureflags.md

```diff
@@ -1,76 +0,0 @@
-
-- `ExperimentalCertificateSigningRequestControllers`. Added in cert-manager
-  1.4.0. Allows to use Kubernetes
-  [CertificateSigningRequest](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/)
-  resources with cert-manager. See [release notes](../releases/release-notes/release-notes-1.4.md#experimental-support-for-kubernetes-certificatesigningrequests)
-
-- `ExperimentalGatewayAPISupport`. Added in cert-manager 1.5.0. Allows to use cert-manager to automatically issue certificates for `Gateway` resources as well as use `Gateway`s and `HTTPRoute`s to solve ACME HTTP-01 challenges. See [Securing Gateway resources](../usage/gateway.md)
-
-- `LiteralCertificateSubject`. Added in cert-manager 1.9.0. Allows to specify certificate subject in a form that can be used to define a location in LDAP directory tree. See [release notes](../releases/release-notes/release-notes-1.9.md#literal-certificate-subjects)
-
-- `ServerSideApply`. Added in cert-manager 1.8.0. If this feature is enabled, cert-manager uses [Server side apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/) when creating or updating API resources. This will speed cert-manager operations and prevent the resource version conflict errors. See [release notes](../releases/release-notes/release-notes-1.8.md#server-side-apply)
-
-- `UseCertificateRequestBasicConstraints`. Added in cert-manager 1.12.0. Makes cert-manager add a basic constraints section to certificate signing requests with the CA constraint set to the correct value. See [`cert-manager#5552`](https://github.com/cert-manager/cert-manager/pull/5552)
-
-- `ValidateCAA`. Added in cert-manager 0.7.2. CAA checking when issuing a certificate.
-
-
-### Beta
-
-These features are enabled by default. See `--feature-gates` flags on cert-manager controller and webhook to disable any of these features.
-
-- `StableCertificateRequestName`. Alpha in v1.10 and Beta in v1.13. Enables generation of `CertificateRequest` resources with a fixed name. See [`cert-manager#5487`](https://github.com/cert-manager/cert-manager/pull/5487)
-
-- `SecretsFilteredCaching`. Alpha in v1.12 and Beta in v1.13. Reduces controller's memory consumption by filtering which Secrets are cached in full using `controller.cert-manager.io/fao` label. By default all Certificate Secrets are labelled with `controller.cert-manager.io/fao` label. Users can also label other Secrets, such as issuer credentials Secrets that they know cert-manager will need access to to speed up issuance. See [`20221205-memory-management.md`](https://github.com/cert-manager/cert-manager/blob/master/design/20221205-memory-management.md)
-
-- `DisallowInsecureCSRUsageDefinition`. Beta in v1.13. Prevents the webhook from allowing CertificateRequest's usages to be only defined in the CSR, while leaving the usages field empty.
```

Comment:
It might have been me that suggested moving this to the contributing section,
but I now think it should be added to the `configuring-components` page, 
so that users can learn about the feature flags available on each component and learn about the maturity of each feature.

Perhaps there should **also** be a section in `contributing/policy` about when it is appropriate to use feature gates, and when not, and link from there to the `installation/component-config` page for a list of current features.

Reply:
I updated the PR. Instead of copying the list of feature gates to the website, I just added a link to the cert-manager code where these gates are defined.

Reply:
The info and the links now live in `installation/component-config`.